src/ontology/fbbt-edit.obo

```diff
@@ -161507,11 +161635,13 @@ creation_date: 2015-05-29T18:36:48Z
 
 [Term]
 id: FBbt:00111062
-name: adult heart circular muscle cell
-def: "Muscle cell of the circular muscle of the adult heart that makes up the wall of the heart. It consists of a single layer, internal to the longitudinal muscle layer." [FlyBase:FBrf0031012]
+name: adult contractile cardiomyocyte
+def: "Contractile cardiomyocyte of the adult dorsal vessel. There are 58 of these cells per dorsal vessel. Their myofilaments are always arranged in a highly regular and circular manner (Lehmacher et al., 2012)." [FlyBase:FBrf0031012, FlyBase:FBrf0219200]
 subset: cur
-is_a: FBbt:00003527 ! adult circular muscle cell
```

Comment:
why are you removing this? It sounds like this muscle is circular?

Reply:
It is, but the definition of `adult circular muscle cell` is a bit too restrictive. It says

> These muscles surrounds [sic] the intestinal tract and ducts of the reproductive system, together with a layer of longitudinal muscle.

Not only is the dorsal vessel not included here, but the “together with a layer of longitudinal muscle” also does not fit: in the case of the dorsal vessel, there’s no longitudinal layer that surrounds the vessel completely, only a _ventral_ longitudinal muscle underneath the ventral side of the vessel.

Do you think we should loosen that definition so that is is applicable to the dorsal vessel?

Reply:
I think this (second sentence) is probably meant as an example rather than a restriction, but it may be confusing, so you can remove/change it if you like.